styles.css

```diff
@@ -22,22 +31,36 @@
     background: #eee;
 }
 
-.grid-cell:nth-child(-n + 3) {
+.game-3x3>.grid-cell:nth-child(-n + 3) {
```

Comment:
can you find another solution for all of this classes rather than this code duplication? maybe use css variables?

Reply:
I actually tried use css variables, but didnt mangae to understand how too use them is this case.
I'll try and check it again